portfolio/src/main/webapp/index.html

```diff
@@ -51,7 +51,7 @@ <h1 class="sub-message">STEP intern summer 2020 for Travel Packages Team</h1>
         <h1 class="sub-message">Worked with fantastic minds on a capstore project</h1>
       </div>
       <div class="headline-sub">
-        <h1 class="sub-message">Ajax | CSS | DataStore | Git | HTML | Java | JavaScript | jQuery | Servlets</h1>
+        <h1 class="sub-message">AJAX | CSS | DataStore | Git | HTML | Java | JavaScript | jQuery | Servlets</h1>
```

Comment:
This looks like changes from another branch, please make sure you push the latest Vision branch to remote. The changes you made in `maps-api` or `Redesign` should already be included, and not be highlighted as a diff here.

Reply:
+1 please fix this. Hard to review otherwise.